main_view.html

```diff
@@ -8,44 +8,324 @@
+                            <div class="framework_img"> <img src="images/middle_section/icon-frameworks.svg" alt=""></div>
+                        
+
+                            <div class="framework_button_action">
+                            <p class="framework_description">Frameworks </p>
+                            
+                            <ul class="framework_buttons">
+                                
+                                     <li class="tools"><button value="Input Buttom"  class="button_frame" type="submit">Bootstrap</button> </li>
+                                     <li class="tools"><button value="Input Buttom"  class="button_frame" type="submit">Ruby on rails</button></li>
+                                     <li class="tools"><button value="Input Buttom"  class="button_frame" type="submit">RSpec</button></li>
+                                     <li class="tools"><button value="Input Buttom"  class="button_frame" type="submit">Capybara</button></li>
+                                     <li class="tools"><button value="Input Buttom"  class="button_frame" type="submit">Selenium</button></li>
+                            </ul>
+                             
+                             </div>
+                           
+                            </div>
+                </div>
+
+                    <div class="skills_main">
+                   
+                            
+                            
+                        <div class="skills_section">
+                            <div class="skills_img"> <img src="images/middle_section/icon-skills.svg" alt=""></div>
+                        
+
+                            <div class="skills_button_action">
+                            <p class="skills_description">Skills</p>
+                            
+                            <ul class="skills_buttons">
+                                
+                                     <li class="tools"><button class="button_skills" value="Input Buttom" type="submit">Codekit</button> </li>
+                                     <li class="tools"><button class="button_skills" value="Input Buttom"  type="submit">GitHub</button></li>
+                                     <li class="tools"><button class="button_skills" value="Input Buttom"  type="submit">Codepen</button></li>
+                                     <li class="tools"><button class="button_skills" value="Input Buttom"  type="submit">Gitlab</button></li>
+                                     <li class="tools"><button class="button_skills" value="Input Buttom" type="submit">Terminal</button></li>
+                            </ul>
+                             
+                             </div>
+                           
+                            </div>
+                </div>
+
+
+
+
+                    <div class="about_main">
+                   
+                                            
```

Comment:
- [ ] Please use a `section` tag instead of `div` tag to follow the HTML semantic tags :blush: due to SEO purpose :+1: